src/index.html

```diff
@@ -13,37 +13,16 @@ <h1 class="whats-cookin" id="whats-cookin">Whats Cookin</h1>
         <button class="nav-btn" id="all-recipes-btn">All Recipes</button>
         <button class="nav-btn" id="favorites-btn">Favorites</button>
         <button class="nav-btn" id="cookbook-btn">Cookbook</button>
-        <select name="tags" class="nav-btn" id="tags">
```

Comment:
Nice job removing the hard coded HTML and implementing functions to add tags dynamically in the scripts.js file.